samples/Team-Import/import_teams_from_custom_source.yaml

```diff
@@ -0,0 +1,86 @@
+metadata:
+  version: "1"
+  dependencies:
+    apps:
+      - id: dynatrace.automations
+        version: ^1.198.1
+      - id: dynatrace.ownership
+        version: ^1.3.2
+  inputs: []
+workflow:
+  title: Import teams from a custom source
+  tasks:
+    import_teams:
+      name: import_teams
+      description: Import teams into Dynatrace.
+      action: dynatrace.ownership:import-teams-to-settings
+      input:
+        importData: '{{ result("fetch_teams_from_external_source") }}'
+        importType: IMPORT_ONLY
+        failStrategy: CONTINUE_AND_LOG_FAILURE
+        importSource: JSON in Ownership schema
+      position:
+        x: 0
+        y: 2
+      predecessors:
+        - fetch_teams_from_external_source
+      conditions:
+        states:
+          fetch_teams_from_external_source: OK
+    fetch_teams_from_external_source:
+      name: fetch_teams_from_external_source
+      description: Build a custom task running js Code
+      action: dynatrace.automations:run-javascript
+      input:
+        script: >-
+          const apiUrl =
+          'https://raw.githubusercontent.com/Dynatrace/Dynatrace-workflow-samples/main/samples/Ownership/Team-Import/example-teams.json';
```

Comment:
Link needs to be fixed: Remove folder "Ownership/"

Reply:
removed.